### PR TITLE
create etc/emacs/run-etags.sh to make argument processing clear

### PR DESCRIPTION
**Pull request by jdchristensen**
@Alizter Let me know what you think about this approach.  I think it's very easy to understand what is going on if it needs to be updated in the future.

### CODE REVIEW

**Comment by Alizter** [comment]
@jdchristensen Are you missing a shell script?

File: Makefile.coq.local [hunk 1]
```diff
@@ -242,7 +240,7 @@ dot-file-dep-graphs: $(ALL_DOTFILES)
 # The TAGS file
 TAGS_FILES = $(ALL_VFILES)
 TAGS : $(TAGS_FILES)
-	$(ETAGS) --language=none $(shell cat etc/emacs/etags-args) $^
+	./etc/emacs/run-etags.sh $^
```
Comment (Alizter) [inline Makefile.coq.local:243]: i.e. where did this come from?

Comment (jdchristensen) [reply Makefile.coq.local:243]: Whoops, fixed.

File: dune [hunk 2]
```diff
@@ -55,7 +55,9 @@
  (alias emacs)
  (mode promote)
  (deps
-  %{bin:etags}
-  (glob_files_rec ./*.v))
+  etc/emacs/run-etags.sh
```
Comment (Alizter) [inline dune:58]: I would keep the `etags` executable here since it is needed for the script.

**Review by Alizter — APPROVED** [review]
LGTM, modulo my comment.

**Comment by Alizter** [comment]
Did you check both make and dune agree btw?

File: dune (hunk 2 above)
Comment (jdchristensen) [reply dune:58]: Right!

**Comment by jdchristensen** [comment]
Yes, they agree, modulo sorting (since the .v files get passed in in a different order).

**Comment by Alizter** [comment]
Great!

**Pull request merged**